katello/db_config.rb

```diff
@@ -0,0 +1,66 @@
+require 'yaml'
+
+module KatelloUtilities
+  class DBConfig
+    attr_reader :foreman, :candlepin
+    INSTALLER_CONFIG = '/etc/foreman-installer/scenarios.d/last_scenario.yaml'
+
+    def initialize(scenario = INSTALLER_CONFIG)
```

Comment:
This is very tight coupling with the puppet modules and that is a risk if we refactor things. I'm not sure if there's a better way though and maybe we should just accept that.